client/src/Pages/Home.js

```diff
@@ -17,17 +18,21 @@ const Home = () => {
         @media (max-width: 480px) {
             background-color: #553d67;
         }
-    `
 
+    `
+    const toSearch = () => {
+        window.location.replace("/search");
```

Comment:
Instead of using `window.location.replace`, I would recommend using a `<Link to='search'>`. The reason for this is a Link object is directly tied into the React router. When you use `window.location.replace`, it doesn't actually reload the page and you are unable to go back to the home page via the browser's back button.

An example of how this is used can be seen on Line 41 inside `DisplayRides.js`